app/products/calls/state/actions.ts

```diff
@@ -446,6 +446,7 @@ export const userReacted = (serverUrl: string, channelId: string, reaction: Call
     } else {
         const newReaction: ReactionStreamEmoji = {
             name: reaction.emoji.name,
+            literal: reaction.emoji.literal,
```

Comment:
Is this going to play nicely against non-updated plugin versions? Not crashing anything that is :)

Reply:
Good question -- yep, it's just an undefined, and that's an optional param for the Emoji component.